app/src/main/java/com/rockspin/androiddevtest/APICommThread.java

```diff
@@ -0,0 +1,33 @@
+package com.rockspin.androiddevtest;
+
+import android.content.Intent;
+
+import java.util.List;
+
+import retrofit2.Response;
+
+/**
+ * Created by SG on 08.01.2017 г..
+ */
+public class APICommThread extends Thread
+{
+    @Override
```

Comment:
I think that for such a small application what you did is fine. Im just warning you that the application we want you to work on uses a lot of libraries that manage things like dependency injection, immutability, posting messages between threads etc.